backend/lib/data/projects_data.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "title": "Loot",
+    "nft_collection": {
+      "collection_url": "https://opensea.io/collection/lootproject"
+    },
+    "smart_contract": {
+      "contract_url": "https://etherscan.io/address/0xff9c1b15b16263c61d017ee9f65c50e4ae0113d7",
+      "address": "0xFF9C1b15B16263C61d017ee9F65C50e4AE0113D7"
+    },
+    "referral_projects": [
+      {
+        "title": "Realms (for Adventurers)",
+        "nft_collection": {
```

Comment:
perhaps rename to `opensea_link`

Reply:
what if url is not OPENSEA? I believe we'll have more projects outside of that platform.  That's why at the beginning I added `Collection link` unstead of `Open Sea link` at the frontend. What if we want to redirect to `Cyberpunks` website for example?

Reply:
ah that makes sense. i think you're right collection_url is better...